### ISSUE TITLE
document available architecture/python versions in introduction of docs

### ISSUE THREAD

**Issue by croth1** [conda-forge/conda-forge.github.io#998]
I think it would be helpful to add a list of currently supported architectures and python/R versions, etc in the  "Brief introduction" section of the docs.
It would be worthwhile to add a note that not all packages are available for all architectures and how users can find out whether their package system is supported by conda-forge.

**Comment by ForgottenProgramme** [comment]
@croth1 , hey! :)
Yes. This makes sense. 
I liked the suggestion that the list of supported architectures should go on the 'brief introduction' page. That way if we do not (yet) support what the user is looking for, their time is saved.

**Comment by ForgottenProgramme** [comment]
Maybe we can add a 'Supported Languages' heading below this : 
https://conda-forge.org/docs/user/introduction.html#display-conda-forge-packages-in-anaconda-navigator heading.

**Comment by ForgottenProgramme** [comment]
@croth1 

@beckermr thinks it would be a good idea to first discuss what we mean by 'support'. 

https://github.com/conda-forge/conda-forge.github.io/issues/1287#issuecomment-806057014 


Also, issue #1287 seems to be a duplicate of this issue.

**Comment by croth1** [comment]
I agree with @beckermr that support needs a qualifier. I think one can liken this to software and plugins. Sometimes we only ship the software (=interpreter/compiler), but not the plugins (=packages).

The section could contain at least following information:
1. What software are we providing and what versions are currently supported.
2. Do we package plugins and if not how can users install 3rd party plugins into their conda environments.
3. If no version is currently supported a link to how people can take over maintainer roles.

What do you think?

**Comment by ForgottenProgramme** [comment]
@croth1 
Yes. Sounds good to me. 
Maybe we could make two sections of what is 'supported'. 

1. Software with packages    
2. Software without packages .

And then a link to 'how to become a maintainer'.

**Comment by viniciusdc** [comment]
> I agree with @beckermr that support needs a qualifier. I think one can liken this to software and plugins. Sometimes we only ship the software (=interpreter/compiler), but not the plugins (=packages).
> 
> The section could contain at least following information:
> 
> 1. What software are we providing and what versions are currently supported.
> 2. Do we package plugins and if not how can users install 3rd party plugins into their conda environments.
> 3. If no version is currently supported a link to how people can take over maintainer roles.
> 
> What do you think?

Also it should be good update this section as well: https://conda-forge.org/docs/orga/guidelines.html?highlight=languages#language-versions

**Comment by viniciusdc** [comment]
Also, as we are talking about the general architecture/package stuff here, it's best to link #986 here too

**Comment by PrernaSingh587** [comment]
@croth1 @viniciusdc 
Hey! I was wondering if I can work on documenting this section. Could you please provide me with some pointers to begin compiling the useful information?

**Comment by beckermr** [comment]
See the discussion above. We cannot make progress on this until we come up with what we want to say in terms of support.

**Comment by viniciusdc** [comment]
yeah, actually I would like to follow @croth1 comments here and break this into different issues (because we might need some big discussions here and there about this: software and versions currently provided by us.) The maintainer role is being discussed somewhere else, so what do you think @beckermr? this way we can start moving forward to the true meaning of support here...

**Comment by beckermr** [comment]
We'll need to chat with all of core. I suggest we add this as an agenda item for tomorrow's meeting

**Comment by Martins-O-U** [comment]
Hello @beckermr and @viniciusdc, Outreachy Applicant here. Please confirm that the "Support" has now been properly defined at your meeting as I'd like to work on documenting this information as a section titled "Supported Languages" on the Brief Introduction page and on the Guidelines section as my contribution.

**Comment by ForgottenProgramme** [comment]
> Hello @beckermr and @viniciusdc, Outreachy Applicant here. Please confirm that the "Support" has now been properly defined at your meeting as I'd like to work on documenting this information as a section titled "Supported Languages" on the Brief Introduction page and on the Guidelines section as my contribution.

Welcome to conda-forge, @Martins-O-U ! :)

**Comment by Martins-O-U** [comment]
Thank you @ForgottenProgramme 
Let's create awesomeness!
